src/driver.ts

```diff
@@ -99,23 +100,50 @@ export const findDevices = (): Device[] => {
 };
 
 /**
- * Turns your Logitech Litra device on.
+ * Turns your Logitech Litra device on
  *
- * @param {Device} device The device to set the temperature of
+ * @param {Device} device The device to turn on
  */
 export const turnOn = (device: Device): void => {
   device.hid.write(padRight([0x11, 0xff, 0x04, 0x1c, 0x01], 20, 0x00));
 };
 
 /**
- * Turns your Logitech Litra device off.
+ * Turns your Logitech Litra device off
  *
- * @param {Device} device The device to set the temperature of
+ * @param {Device} device The device to turn off
  */
 export const turnOff = (device: Device): void => {
   device.hid.write(padRight([0x11, 0xff, 0x04, 0x1c, 0x00], 20, 0x00));
 };
 
+/**
+ * Toggles your Logitech Litra device on or off
+ *
+ * @param {Device} device The device to toggle on or off
+ */
+export const toggle = (device: Device): void => {
+  if (getPowerState(device)) {
+    turnOff(device);
+  } else {
+    turnOn(device);
+  }
+};
+
+/**
+ * Gets the current power state of your Logitech Litra device
+ *
+ * @param {Device} device The device to get the current power state for
+ * @returns {boolean} Current power state where true = on and false = off
+ */
+export const getPowerState = (device: Device): boolean => {
```

Comment:
Can we rename this to `isOn`? I feel like that name fits nicely, and the API design of returning a boolean makes a lot of sense with that name.

```suggestion
export const isOn = (device: Device): boolean => {
```

After that change, we'd want to tweak the function documentation to align with it.

Reply:
The other functions have a verb as a prefix (like get, set, turn, etc..). Would it make more sense to call it `checkIfOn` or something similar for consistency? `isOn` doesn't sound like a function to me and if I were using the API I'd be a little surprised. Nonetheless it's your call and not a big deal, let me know!

Reply:
That's a good point about the normal use of verbs.\

If it's `getPowerState`, I'd expect it to return `'on'` or `'off'`, but really a boolean makes sense in this case.

I think I'd still prefer to go for `isOn`, if you don't mind. It just feels like the most natural option to me!